### PR TITLE
Move gas amount constants to BlockchainConfigs

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Move gas amount constants to BlockchainConfigs
- Increase gas amounts (REST_FUNCTION_CALL_GAS_AMOUNT, ACCOUNT_REGISTRATION_GAS_AMOUNT)
- Use UNIT_WRITE_GAS_AMOUNT instead of 1

### CODE REVIEW

File: common/constants.js [hunk 1]
```diff
@@ -142,6 +142,9 @@ BlockchainConfigs.SERVICE_TREE_SIZE_BUDGET = BlockchainConfigs.SERVICE_STATE_BUD
 BlockchainConfigs.APPS_TREE_SIZE_BUDGET = BlockchainConfigs.APPS_STATE_BUDGET * BlockchainConfigs.MAX_STATE_TREE_SIZE_PER_BYTE;
 BlockchainConfigs.FREE_TREE_SIZE_BUDGET = BlockchainConfigs.FREE_STATE_BUDGET * BlockchainConfigs.MAX_STATE_TREE_SIZE_PER_BYTE;
 BlockchainConfigs.STATE_GAS_COEFFICIENT = 1;
+BlockchainConfigs.UNIT_WRITE_GAS_AMOUNT = 1;
```
Comment (liayoo) [inline common/constants.js:145]: Thanks for introducing this constant! 👍

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

File: common/constants.js (hunk 1 above)
Comment (minsulee2) [reply common/constants.js:145]: 👍

**Review by minsulee2 — APPROVED** [review]
LGTM!

**Pull request merged**